pkg/metrics/resource.go

```diff
@@ -67,6 +69,16 @@ var (
 		Buckets: prometheus.ExponentialBuckets(0.001, 2, 12),
 	}, []string{"result"})
 
+	reCreateResourceWhenSyncWorkStatus = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: reCreateResourceToCluster,
+		Help: "Number of re-creating operation of the resource to a target member cluster. By the result, 'error' means a resource recreated failed. Otherwise 'success'. Kind means the resource kind. Cluster means the target member cluster.",
+	}, []string{"result", "kind", "cluster"})
```

Comment:
Would it be better to replace kind with gvk?

Reply:
done